src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -370,22 +392,22 @@ Status GcsActorManager::RegisterActor(
   if (iter != registered_actors_.end() &&
       iter->second->GetState() == rpc::ActorTableData::ALIVE) {
     // When the network fails, Driver/Worker is not sure whether GcsServer has received
-    // the request of actor creation task, so Driver/Worker will try again and again until
-    // receiving the reply from GcsServer. If the actor has been created successfully then
-    // just reply to the caller.
+    // the request, so Driver/Worker will try again and again until receiving the reply
+    // from GcsServer. If the actor has been created successfully then just reply to the
+    // caller.
```

Comment:
// In case of temporary network failures, workers will re-send multiple duplicate requests to GCS server. 
// In this case, we can just reply.